smart-note/e2e/addnote/addnote.e2e-spec.ts

```diff
@@ -1,22 +1,55 @@
-// import { Addnote } from './addnote.po';
-// import { browser } from 'protractor';
+import { Addnote } from './addnote.po';
+import { by, browser, protractor } from 'protractor';
 
-// // sleep for demonstration 
-// function sleep() {
-//     browser.driver.sleep(2000); 
-//   }
+// sleep for demonstration 
+function sleep() {
+    browser.driver.sleep(3000); 
+  }
+  
+var selectDropdownbyNum = function ( element, optionNum ) {
+    if (optionNum){
+        var options = element.findElements(by.css('my-dropdown-menu'))   
+        .then(function(options){
+            options[optionNum].click();
+        });
+    }
+};
 
-// describe('Addnote page', () => {
-//   let page: Addnote;
+describe('Addnote page', () => {
```

Comment:
Good job, this fully test the addnote component, but some of the edge cases are not test yet, like saving a note without adding a tag, saving a note without adding any content, etc.